pom.xml

```diff
@@ -657,6 +657,116 @@
             </build>
         </profile>
 
+        <profile>
+            <id>payara-micro-remote</id>
+
+            <properties>
+                <!-- default port -->
+                <payara.httpPort>8080</payara.httpPort>
+                <!-- JMS and JAXWS not supported by Micro -->
+                <skipJMS>true</skipJMS>
+                <skipJAXWS>true</skipJAXWS>
+
+                <!-- Being a WebProfile++, Micro does not support ear archives -->
+                <skipEAR>true</skipEAR>
+
+                <!-- Client-cert needs complicated cert setup, which hasn't been done -->
```

Comment:
"which hasn't been done". Is that out of scope, or a planned future change?

Reply:
Mostly, it's a copy-paste from managed profile. But also I wouldn't consider it in scope if managed profile cannot do it.

Reply:
Improved the wording